x-pack/plugins/reporting/server/export_types/common/get_full_urls.test.ts

```diff
@@ -30,7 +30,8 @@ beforeEach(async () => {
 const getMockJob = (base: object) => base as TaskPayloadPNG & TaskPayloadPDF;
 
 test(`fails if no URL is passed`, async () => {
-  const fn = () => getFullUrls(mockReporting, getMockJob({}));
+  const fn = () =>
+    getFullUrls(mockReporting.getServerInfo(), mockReporting.getConfig(), getMockJob({}));
```

Comment:
Same as above: the server info and config should be mocked without a mockReportingCore